style.css

```diff
@@ -0,0 +1,319 @@
+    width: 40%;
+}
+
+.skills-p{
+    margin-top: 2em;
+    color: lightgoldenrodyellow;
+    letter-spacing: 3px;
+}
+
+li{
+    list-style: none;
+}
+
+.skills{
+    display: flex;
+    justify-content: space-around;
+}
+
+.skill-set{
+    margin-top: 1em;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.skill{
+    margin-top: 2em
+}
+
+.front-end{
+    /* background-color: palegoldenrod; */
+    height: 50vh;
+    width: 45%;
+}
+
+.back-end{
+    /* background-color: palegoldenrod; */
+    height: 50vh;
+    width: 45%;
+}
+
+#soon{
+    color: rgb(250,250,210);
+    letter-spacing: 3px;
+}
+
+
+#experience{
+    height: 103vh;
+    background-color: darkolivegreen;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.experiences{
+    display: flex;
+    justify-content: space-evenly;
+    flex-direction: row;
+    align-items: center;
+}
+
+.experience{
+    background-color: #f7f7f7;
+    padding: 2em;
+    width: 30%;
+    height: 30em;
+    padding: 0 0 100px 0;
+    border-radius: 1em;
+    overflow: hidden;
+    margin-bottom: 1em;
+    box-shadow: 0 0 1px lightgrey;
+}
+
+.experience h1{
+    padding: 1em;
+    color: #4d61f4;
+}
+
+.experience p{
+    padding: 2em;
+    color: #4d61f4;
+}
+
+#projects{
+    background-color: #4d61f4;
+
+    height: auto;
+}
+
+#projects h1{
+    padding: 3em;
+    text-align: center;
+    letter-spacing: 5px;
+    color: lightgoldenrodyellow;
+    margin: -55px;
+    font-size: 2em;
+}
+
+.projects{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-top: 50px;
+}
+
+.project{
+    height: 30%;
+    display:flex;
+    justify-content: space-around;
+    margin-top: 30px
+}
+
+.content{
+    width: 40%;
+}
+
+.content h1{
+    align-self: center;
+}
+
+.buttons-controller{
+    display: flex;
+    justify-content: space-evenly;
+    margin-top: 20px;
+    margin-bottom: 10px;
+    align-self: center;
+}
+
+.description{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 30%;
+    text-align: center;
+}
+
+
+.description p{
+    color: lightgoldenrodyellow;
+}
+
+.skill-set-projects{
+    padding: 1em;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+.footer{
+    height: 2em;
+    padding: 2em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: darkolivegreen;
+}
+
+.footer p{
+    color: lightgoldenrodyellow;
+}
+
+.hr{
+    margin-top: 10px;
+    width: 85%;
+    align-self: center;
+}
+
+.slider{
+    width: 100%;
+    height: 500px;
+    margin: auto;
+    box-shadow: 0px 0px 10px grey;
+    position: relative;
+    overflow: hidden;
+}
+
+.slide{
+    width: 100%;
+    height: 100%;
+    transition: 1s;
+    position: absolute;
+}
+
+.buttons-controller{
+    height: 1em;
+}
+
```

Comment:
Naming conventions for attributes are clear, logical, and easy to follow.  Also, effective use and understanding with slightly more specific selectors in the elements that you're targeting.